src/3rd_party-static/encryption/src/Base64.cpp

```diff
@@ -82,7 +82,7 @@ std::string base64_encode(unsigned char const* bytes_to_encode, unsigned int in_
 }
 
 std::string base64_decode(std::string const& encoded_string) {
-  int in_len = encoded_string.size();
+  int in_len = (int)encoded_string.size();
```

Comment:
I don't think I have to change even variables type in 3rd-party library. I'm going to use static_cast here